openadapt/scrub.py

```diff
@@ -265,29 +228,19 @@ def scrub_dict(
     Returns:
         dict: The scrubbed dict with PII and PHI removed.
     """
-
-    if config.SCRUB_ENABLED is False:
-        return input_dict
-
     if list_keys is None:
         list_keys = config.SCRUB_KEYS_HTML
 
     scrubbed_dict = {}
     for key, value in input_dict.items():
         if _should_scrub_text(key, value, list_keys, scrub_all):
-            scrubbed_text = _scrub_text_item(
-                value, key, force_scrub_children
-            )
-            if key in ("text", "canonical_text") and _is_scrubbed(
-                value, scrubbed_text
-            ):
+            scrubbed_text = _scrub_text_item(value, key, force_scrub_children)
+            if key in ("text", "canonical_text") and _is_scrubbed(value, scrubbed_text):
```

Comment:
I find it more readable to keep one statement per line. Is the the default in `black`? Any way to change it?

Reply:
I think it may have originally been on one line, but my black formatter does this.

Reply:
manually corrected it in scrub.py for now